packages/oats/src/generate-server.ts

```diff
@@ -498,6 +498,42 @@ export function generateCreateRouter() {
                         ]),
                         ts.factory.createObjectLiteralExpression()
                       ),
+                      ts.factory.createBindingElement(
+                        undefined,
+                        'body',
+                        ts.factory.createObjectBindingPattern([
+                          ts.factory.createBindingElement(
+                            undefined,
+                            'unknownField',
+                            'bodyUnknownField',
+                            ts.factory.createStringLiteral('fail')
+                          ),
+                          ts.factory.createBindingElement(
+                            undefined,
+                            'parseBooleanStrings',
+                            'bodyParseBooleanStrings',
+                            ts.factory.createTrue()
+                          ),
+                          ts.factory.createBindingElement(
+                            undefined,
+                            'parseNumericStrings',
+                            'bodyParseNumericStrings',
+                            ts.factory.createTrue()
+                          ),
+                          ts.factory.createBindingElement(
+                            undefined,
+                            'allowConvertForArrayType',
+                            'bodyAllowConvertForArrayType',
+                            ts.factory.createTrue()
+                          ),
```

Comment:
i don't think we need to deconstruct these

Reply:
Yeah, no need at least for now. Removed 👍